CHANGELOG.md

```diff
@@ -1,5 +1,8 @@
 # [Changelog](https://github.com/yola/drf-madprops)
 
+## 0.3.0
```

Comment:
Hmm... I see that we have django requirement in setup.py (but not in this pull). That's probably the reason for version bump.